### DISCUSSION TITLE
What does this drive?

### DISCUSSION THREAD

**Discussion by baysinger** [zwave-js/zwave-js#3087] [Q&A] [answered]
Sorry for this basic-as-can-be question, but coming here with little context, the answer is not obvious. The readme says it's a "Z-Wave driver written entirely in JavaScript/TypeScript". Ok, but what does it drive?

In the intro, it says it "runs on almost anything with a little bit of computing power and a serial port." So that suggests it a driver for some sort of z-wave serial device. But what serial z-wave device does it work with?

**Comment by AlCalzone** [comment, marked as answer]
It controls Z-Wave USB sticks and/or Raspberry Pi hats like these:

- https://aeotec.com/z-wave-usb-stick/
- https://aeotec.com/z-wave-usb-stick/z-stick-7.html
- https://aeotec.com/z-wave-home-automation/development-kit-pcb.html-
- https://www.getzooz.com/zooz-zst10-s2-stick.html
- and all the others that are available

Since they all speak the same protocol, they should all work out of the box.

**Reply by baysinger** [reply]
Thanks for the answer! I think this little bit of info would be good to add to the readme. The context that I was missing is that all Z-Wave USB sticks speak the same protocol.

**Reply by AlCalzone** [reply]
Would you be willing to provide that context with a PR? I can sometimes be a bit routine-blinded after working so long on this.

**Reply by Hedda** [reply]
Suggest https://github.com/zwave-js/node-zwave-js/pull/3175

**Reply by baysinger** [reply]
I'd been meaning to get back here and make a PR, but looks like @Hedda beat me to it! (I just started a new job last week, so time has been short.)

**Reply by AlCalzone** [reply]
PR merged, thanks all
